pages/api/plugins/debugPlugin.ts

```diff
@@ -0,0 +1,21 @@
+export const DebugPlugin = {
```

Comment:
I'm assuming that we don't need this long term

Reply:
In terms of teaching a customer how to developer with GraphQL, I think its an interesting tool.